internal/arg/arg.go

```diff
@@ -0,0 +1,17 @@
+package arg
+
+import (
+	"errors"
+	"fmt"
+)
+
+// ValidateCount validates the arguments are of expected length.
+func ValidateCount(args []string, expLen int, missingErrMsg string) error {
```

Comment:
nit:
I think this function should be renamed to `ValidateArgsCount`

Reply:
the usage would be `arg.ValidateCount`. IMO adding args to function name is redundant.

Reply:
Oh yeah, that's true.